### PR TITLE
Modify docs to include LX

### PR DESCRIPTION
**Pull request by joelmozhu**
See: https://bnc.tpondemand.com/entity/6722-update-documentation-for-index-ticker-and

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -194,7 +196,7 @@ components:
           type: string
           format: date-time
         id:
-          description: Unique identifier for the resource.
+          description: Unique identifier for the constituent used to calculate the index.
```
Comment (alphafoobar) [inline openapi.yaml]: The id is the `Unique identifier for the resource`. Please revert this change.

File: openapi.yaml [hunk 2]
```diff
@@ -1374,21 +1387,25 @@ paths:
...
           in: query
           description: |
             Retrieve all the indices by the particular index id. The index id will differ based on the index type.
 
-            | Index Type | Index Id  |
-            | -----------|:---------:|
-            | MWA        |Market Id  |
-            | GWA        |Asset Id   |
+            | Index Type | Index Id  |  Example Id | Description |
+            | -----------|:---------:|:-----------:|             |
```
Comment (alphafoobar) [inline openapi.yaml]: missing `---`

File: openapi.yaml [hunk 3]
```diff
@@ -1346,13 +1355,17 @@ paths:
...
 
         Each index type will have an index id which identifies the constituent type used to build the index -
 
         | Index Type | Index Id  | Output |
         | -----------|:---------:|:---------:|
         | MWA        | Market Id | The Market Weighted Average for that market |
         | GWA        | Asset Id  | The Global Weighted Average for that coin / asset |
+        | LX         | Index Id  | Bitcoin Liquid Index (BLX)
+        | LX         | Index Id  | Ethereum Liquid Index (ELX)
+        | LX         | Index Id  | Ripple Liquid Index (XRPLX)
```
Comment (alphafoobar) [inline openapi.yaml]: We can use the actual UUIDs for the BLX, ELX, XRPLX here. See https://docs.bravenewcoin.com/#tag/Websocket-LX though that uses an `Example Id`

Also missing end column.

File: openapi.yaml [hunk 4]
```diff
@@ -177,7 +178,7 @@ components:
       type: object
       properties:
         indexId:
-          description: Unique identifier for the constituent used to calculate the index.
+          description: Unique identifier for the resource.
```
Comment (alphafoobar) [inline openapi.yaml]: `Unique identifier for the constituent used to calculate the index.`. Is correct for GWA and MWA indices, we could add `Unique identifier for the constituent used to calculate the index. Or a unique ID for the index in the case of LX indices.`?

File: openapi.yaml (hunk 2 above)
Comment (alphafoobar) [reply openapi.yaml]: Maybe just collapse Example Id into index Id for the LX indices?

File: openapi.yaml [hunk 5]
```diff
@@ -1446,18 +1464,24 @@ paths:
...
 
         Each index type will have an index id which identifies the constituent type used to build the index -
 
         | Index Type | Index Id  | Output |
         | -----------|:---------:|:---------:|
         | MWA        | Market Id | OHLCV for that market |
         | GWA        | Asset Id  | OHLCV for that coin / asset |
+        | LX         | Index Id  | OHLCV for Bitcoin Liquid Index (BLX)
+        | LX         | Index Id  | OHLCV for Ethereum Liquid Index (ELX)
+        | LX         | Index Id  | OHLCV for Ripple Liquid Index (XRPLX)
```
Comment (alphafoobar) [inline openapi.yaml]: Missing end column for LX, maybe use a standard template?

File: openapi.yaml [hunk 6]
```diff
@@ -1470,21 +1494,26 @@ paths:
...
-            | -----------|:---------:|
-            | MWA        |Market Id  |
-            | GWA        |Asset Id   |
+            | Index Type | Index Id  | Example Id | Description |
+            | -----------|:---------:|:----------:|             |
+            | MWA        |Market Id  |            |             |
+            | GWA        |Asset Id   |            |             |
+            | LX         |Index Id   | 191724d4-7915-491b-8b73-bc293e03a93e | [BLX](https://bravenewcoin.com/data-and-charts/indices/blx) |
+            | LX         |Index Id   | 6560acf3-1cb7-4fe4-aa19-fbd90c7b56ee | [ELX](https://bravenewcoin.com/data-and-charts/indices/elx) |
+            | LX         |Index Id   | ce36445a-2a33-4a0f-8b5b-e56175f8db60 | [XRPLX](https://bravenewcoin.com/data-and-charts/indices/blx) |
```
Comment (alphafoobar) [inline openapi.yaml]: Collapse together?

Comment (alphafoobar) [reply openapi.yaml]: Or another alternative is to have a section for LX indexes or just indexes in general that we reference instead of duplicating this Index section everywhere?

**Review by alphafoobar — CHANGES_REQUESTED** [review]
Let's catch up and review this before merging, you have identified a few descriptions that are difficult to follow and highlighted that there is a lot of duplication through the documentation.

File: openapi.yaml (hunk 5 above)
Comment (joelmozhu) [reply openapi.yaml]: Done

File: openapi.yaml (hunk 4 above)
Comment (joelmozhu) [reply openapi.yaml]: Done

**Comment by alphafoobar** [comment]
Approved.
